emain/emain-window.ts

```diff
@@ -699,11 +701,16 @@ ipcMain.on("delete-workspace", (event, workspaceId) => {
             console.log("user cancelled workspace delete", workspaceId, ww?.waveWindowId);
             return;
         }
-        await WorkspaceService.DeleteWorkspace(workspaceId);
+        
+        const moveToNewWorkspace = await WorkspaceService.DeleteWorkspace(workspaceId) 
         console.log("delete-workspace done", workspaceId, ww?.waveWindowId);
-        if (ww?.workspaceId == workspaceId) {
-            console.log("delete-workspace closing window", workspaceId, ww?.waveWindowId);
-            ww.destroy();
+        if (ww?.workspaceId == workspaceId){
+            if ( workspaceList?.length > 1 ) {
```

Comment:
this should check if the new workspace id is not null, the workspaceList check should no longer be necessary since the backend is dealing with this